### ISSUE TITLE
two out of three voting sites - working

### ISSUE THREAD

**Issue by smmmadden** [BenCodez/VotingPlugin#243]
Just installed the latest update and it fixed the CraftServers voting so it now registers in-game.
![image](https://user-images.githubusercontent.com/10309346/29104257-6f4769d8-7c92-11e7-9e26-a37b101fbbe9.png)
The minecraftforum.net site also works, but the very first one I tried with minecraft-mp.com still doesn't get registered back.  So I'm wondering if this is just a problem with them not handling the NuVotifier plugin?

I double checked the totals issue and that is fixed, but a new problem shows the bow (left most slot) pushes it off-screen to the left and not visible.
![image](https://user-images.githubusercontent.com/10309346/29104325-c136559c-7c92-11e7-9814-4d3798a5c60b.png)

**Comment by BenCodez** [comment]
My first guess is a votifier issue.

The offscreen to the left is just how minecraft does it, nothing I can really do.

**Comment by smmmadden** [comment]
that's really odd.  Users can't see where they vote for in the first slot? Seems like it isn't using the same display as all the others where it hovers to the right of the slot selected.

I turned on debugging to see why totals aren't showing and this is what it shows:
[21:45:20] [Craft Scheduler Thread - 17/INFO]: [VotingPlugin] Debug: Starting background task
[21:45:20] [Server thread/INFO]: [0;33;1m[Jobs] Successfully linked with Vault.[m
[21:45:20] [Craft Scheduler Thread - 17/INFO]: [VotingPlugin] Debug: Finished loading player data
[21:45:20] [Craft Scheduler Thread - 17/INFO]: [VotingPlugin] Debug: Extra: Failed to get int from 'AllTimeTotal' for 'minecraftforum'
[21:45:20] [Craft Scheduler Thread - 17/INFO]: [VotingPlugin] Debug: All Time TopVoter loaded
[21:45:20] [Craft Scheduler Thread - 17/INFO]: [VotingPlugin] Debug: Extra: Failed to get int from 'MonthTotal' for 'minecraftforum'
[21:45:20] [Craft Scheduler Thread - 17/INFO]: [VotingPlugin] Debug: Monthly TopVoter loaded
[21:45:20] [Craft Scheduler Thread - 17/INFO]: [VotingPlugin] Debug: Updated TopVoter
[21:45:20] [Craft Scheduler Thread - 17/INFO]: [VotingPlugin] Debug: Updated VoteToday
[21:45:20] [Craft Scheduler Thread - 17/INFO]: [VotingPlugin] Debug: Signs updated
[21:45:20] [Craft Scheduler Thread - 17/INFO]: [VotingPlugin] Debug: Background task ran

**Comment by BenCodez** [comment]
I think you can customize the text to make it smaller in the Config.yml.

You got no all time total?

**Comment by smmmadden** [comment]
Sorry for delay getting back to you, a lot of plugins were updated this week that sucked up my time.
I've updated all the plugins, the server code and bungee.  I removed the minecraft-mp.com site for voting as it wasn't truly supporting nuvotifier/votingplugin.   The other two are now recording totals and other details for voting on them.

To fix it, it appears that each of the worlds config.yml had something wrong in it.  So I started from scratch and restarted.  I also enabled the rewards for each server using the examples you provided.  I tried to rename the two examples, but on server startup, the files are recreated and actually loaded with the plugin which to me didn't make sense as they are examples and when the rewards are created for each voting server a separate file is created for that server.  So no sense in loading data into memory when it won't be used.  Other than that, I'd say it is finally safe to close this issue. :-)
